webserver/src/main/java/io/kestra/webserver/controllers/PluginController.java

```diff
@@ -33,6 +36,9 @@
 @Validated
 @Controller("/api/v1/plugins/")
 public class PluginController {
```

Comment:
Please preserve original indent in this file.
Continuous ident is 4 not 8.

Reply:
It gets reset all the time idk why...